sys/linux/socket_netlink_netfilter_nftables.txt

```diff
@@ -19,7 +19,7 @@ include <net/xfrm.h>
 type msghdr_nf_tables[CMD, POLICY] msghdr_netlink[netlink_msg_netfilter_t[NFNL_SUBSYS_NFTABLES, CMD, POLICY]]
 # TODO: we should obtain them from somewhere, probably from other netlink messages,
 # but we can't extract output netlink attributes.
-type nft_chain_id int32be
+type nft_chain_id int32be[1:4]
```

Comment:
May nft_chain_id really really have only such a small range of values?

Reply:
> May nft_chain_id really really have only such a small range of values?

int32 ids would hardly match any existing chain since they're user-provided with no real logic behind them. Shall we raise it to 100 to keep a decent amount of possible chains while also being able to lookup by id?

Reply:
Ack, let's just limit it like this.
Maybe leave some comment about why we have restricted it and that `1:4` does not come from anywhere in the kernel?